src/main/java/seedu/application/logic/commands/AddCommand.java

```diff
@@ -34,7 +32,8 @@ public class AddCommand extends Command {
         + PREFIX_COMPANY + "Google"
         + PREFIX_ROLE + "Software Engineer "
         + PREFIX_DEADLINE + "Dec 31 2030 1200"
-        + PREFIX_STATUS + "pending\n";
+        + PREFIX_STATUS + "pending\n"
+        + PREFIX_INDUSTRY + "Healthcare";
```

Comment:
Maybe can provide more realistic example here? For example if our previous example is SWE role at Google, maybe we should indicate the industry as "Tech" or similar?